### ISSUE TITLE
Rename the core module

### ISSUE THREAD

**Issue by matthewcarbone** [AI-multimodal/aimm-post-processing#1]
`aimm_post_processing` is a bit of a mouth full when doing imports. Thoughts? Maybe `ap2` for "AIMM post processing"? 😄

**Comment by zleung9** [comment]
I agree. We can keep the repo name while changing the project name (the folder within aimm-post-procssing) to something shorter, ap2?

**Comment by matthewcarbone** [comment]
@zhul9311 Yes we can. The question is, should we... it would be nice to have @danielballan and @jmaruland feedback on this, but it's really not a priority issue.
